python/sglang/bench_latency.py

```diff
@@ -267,13 +297,25 @@ def run_once(output_len):
         rank_print(
             f"Total. latency: {tot_latency:6.3f} s, throughput: {throughput:9.2f} token/s"
         )
+        measurement_results["total_latency"] = tot_latency
+        measurement_results["total_throughput"] = throughput
+        return measurement_results
 
     # Warm up
     run_once(4)
     clear()
 
     # Run again
-    run_once(bench_args.output_len)
+    ret = []
+    ret.append(run_once(bench_args.output_len))
+
+    # Write results
+    if bench_args.result_filename:
+        data = []
+        if os.path.isfile(bench_args.result_filename):
+            data = json.loads(open(bench_args.result_filename, "r").read())
+        data += ret
+        open(bench_args.result_filename, "w").write(json.dumps(data, indent=2))
```

Comment:
Can we use jsonl format with append mode?

Reply:
Great idea. Will do.

Reply:
resolved in the new commit https://github.com/sgl-project/sglang/pull/909/commits/f52110d52b4b96ced2dbd25dadc8c90d99af5ad6